initfiles/systemd/gitautodeploy.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=GitAutoDeploy
+
+[Service]
+User=root
```

Comment:
That is also wrong, sadly. This process shouldn't never run as root, but as it's own user/group.